module/x/gravity/abci.go

```diff
@@ -100,12 +102,30 @@ func pruneSignerSetTxs(ctx sdk.Context, k keeper.Keeper) {
 		earliestToPrune := currentBlock - params.SignedSignerSetTxsWindow
 		for _, set := range k.GetSignerSetTxs(ctx) {
 			if set.Nonce < lastObserved.Nonce && set.Height < earliestToPrune {
+				k.DeleteEthereumSignatures(ctx, set.GetStoreIndex())
```

Comment:
Since we always use a combination of `DeleteEthereumSignatures` and `DeleteOutgoingTx`, should we just move `DeleteEthereumSignatures` into `DeleteOutgoingTx`?